src/posts/2021-06-14-Everyday_Accessibility.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: Everyday Accessibility
+description: "Make your social media, digital document, and multimedia content more accessible with these quick and practical steps."
+category: Quick tip
+author: Michele A. Williams, PhD
+date: 2021-06-15
+eleventyNavigation:
+  key: { { category } }
+  title: { { title } }
+tags: quicktip
+---
+
+Even with the best of intentions, you may be making decisions that exclude disabled users. Below are practical, intentional steps you can start today to create and share more accessible social media, digital, and multimedia content.
+
+## Social Media Posts
+
```

Comment:
Can we add a bridge sentence here that queues up the content in the subheadings? I've been told that without it there can be confusion if content is missing/failed to load, or the heading structure is broken somehow.

Reply:
Yes, @mxmason and I were just discussing this. Initially I thought I could sneak by when I had these as numbered lists but now that they're headings I'll add a little something. I just hate when it's sounds too redundant (e.g., waste of reading time) so I'll see what _useful_ content I can come up with so it's not drab...